x/metadata/keeper/msg_server.go

```diff
@@ -48,6 +59,44 @@ func (k msgServer) WriteScope(
 	return types.NewMsgWriteScopeResponse(msg.Scope.ScopeId), nil
 }
 
+func (k msgServer) createExpirationForScope(ctx sdk.Context, msg *types.MsgWriteScopeRequest) (*exptypes.Expiration, error) {
+	// create expire scope request.
+	expireMsg := types.NewMsgExpireScopeRequest(msg.Scope.ScopeId, msg.Signers)
+	wrapper, anyErr := codec.NewAnyWithValue(expireMsg)
+	if anyErr != nil {
+		return nil, anyErr
+	}
+
+	// use default expiration, but override with cmd line arg if provided
+	expShort := k.GetDefaultScopeExpiration(ctx, msg.Scope)
+	if len(msg.Expiration) > 0 {
+		expShort = msg.Expiration
+	}
+
+	// parse expiration into a duration
+	duration, durErr := exptypes.ParseDuration(expShort)
+	if durErr != nil {
+		return nil, durErr
+	}
+
+	// REVIEW THIS: for now, using first owner for scope as expiration depositor
```

Comment:
Without adding a field to the `MsgWriteScopeRequest`, I think using the first owner is the best way to go.

Reply:
updated comment to `using first owner for scope as expiration depositor`